dotnet/src/SemanticKernel/CoreSkills/SemanticFunctionConstants.cs

```diff
@@ -77,7 +76,7 @@ internal static class SemanticFunctionConstants
     <function._GLOBAL_FUNCTIONS_.SendEmail input=""$TRANSLATED_SUMMARY"" email_address=""$EMAIL_ADDRESS""/>
 </plan><!-- END -->
 
-[AVAILABLE FUNCTIONS]
+[EXAMPLE FUNCTIONS]
```

Comment:
Do you find this improves things to help differentiate the pattern?

Reply:
Yes, I was getting very frequently WriterSkill.Summarization instead of the Everything, this was happening with Skills that bumped in the examples given and for some reason the LLM was understanding AVAILABLE examples as indeed available...

Reply:
Something for us to consider when we start allowing users to supply their own examples.

Reply:
Even changing that to this pattern I see less frequently it using the wrong function, potentially we should use abstract examples that would not conflict with actual skills.